packages/venia-ui/lib/components/Header/header.css

```diff
@@ -1,3 +1,7 @@
+:root {
+    --header-height: 5rem;
```

Comment:
Do we define other global variables in such a way? I'd like to talk about this in dev time.

cc @jimbo

Reply:
Change to be header local variable and value is set in header component and depends on header inner layout.

Reply:
Ah I see. You did it like this:

```
const style = {
        ...(switchers && { '--header-height': '7.5rem' })
    };

    return (
        <header className={rootClass} style={style}>
        ...
```